src/components/Data/Layout.vue

```diff
@@ -67,6 +71,10 @@ export default {
     }
   },
   methods: {
+    registerNewPaneSize(size) {
```

Comment:
The verb `register` in function name, conventionally means associating a function (coming from outside the scope of the program) to a hook, or something similar. I think the verb you want to use here is "save".